### PR TITLE
Add electrs logo

### PR DESCRIPTION
**Pull request by romanz**
Provided by Dominik Průša under the MIT license.

### CODE REVIEW

**Comment by romanz** [comment]
Many thanks for @Kixunil for the [idea](https://github.com/romanz/electrs/issues/481) and the help!

**Comment by romanz** [comment]
Added the logo to README.md here :)
https://github.com/romanz/electrs/blob/logo/README.md

**Comment by Kixunil** [comment]
Looks beautiful at the top of README! I think it makes sense to publish manual too.

**Comment by Kixunil** [comment]
Dominik has GitHub profile @DominoPrusa, let's link it.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![Logo](https://raw.githubusercontent.com/romanz/electrs/master/logo/logo.svg)
```
Comment (Kixunil) [inline README.md:1]: Is there any issue with relative path? Absolute will look broken until merged.

Comment (romanz) [reply README.md]: TIL relative paths work for images :)

**Comment by Kixunil** [comment]
ACK :)

**Pull request merged**